matminer/featurizers/site.py

```diff
@@ -33,6 +36,18 @@
 
 from matminer.featurizers.stats import PropertyStats
 
+cn_motif_op_paras = {}
```

Comment:
note that we usually abbreviate as ``params`` rather than ``paras``. Params is used >1200 times in the various repos, paras is used ~125 times, mostly by the local env code